activemq/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,23 @@
+name: ActiveMQ
+files:
+- name: activemq.yaml
+  options:
+  - template: instances
+    options:
+    - template: instances/jmx
+    - template: instances/default
+  - template: logs
+    example:
+    - type: file
+      path: <ACTIVEMQ_BASEDIR>/data/activemq.log
+      source: activemq
+      service: <SERVICE_NAME>
+    - type: file
+      path: <ACTIVEMQ_BASEDIR>/data/audit.log
+      source: activemq
+      service: <SERVICE_NAME>
+  - template: init_config
+    options:
+      - template: init_config/jmx
+      - template: init_config/default
```

Comment:
Most integrations use this order:

- init_config
- instances
- logs

Reply:
I know but I wanted to keep the diff on the existing config as small as possible. I'm ok with changing it to be more standard